tests/test_main.py

```diff
@@ -12,19 +14,17 @@
         ("Abracadabra", "A", 5),
```

Comment:
This test case ('Abracadabra', 'A', 5) seems to be incorrect. The letter 'A' appears only twice in the phrase 'Abracadabra' if we consider case sensitivity.